pkg/vaildations/cluster_version.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const (
-	clusterVersionID  = "Opensift Cluster Version is valid"
+	clusterVersionID  = "https://github.com/redhat-partner-solutions/vse-sync-test/tree/main/tests/sync/G.8272/environment/version/ocp/testspec.adoc"
```

Comment:
```suggestion
	clusterVersionID  = "https://github.com/redhat-partner-solutions/vse-sync-test/tree/main/tests/sync/G.8272/environment/version/ocp/"
```

Reply:
testspec.adoc isn't (formally) part of the test URI

Reply:
Correct, removed